tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -320,6 +320,26 @@ public function testBooleanDefault(callable $comparatorFactory): void
         self::assertFalse($diff);
     }
 
+    /**
+     * @param callable(AbstractSchemaManager):Comparator $comparatorFactory
+     *
+     * @dataProvider \Doctrine\DBAL\Tests\Functional\Schema\ComparatorTestUtils::comparatorProvider
+     */
+    public function testGeneratedColumn(callable $comparatorFactory): void
```

Comment:
@derrabus  this test fail on version of postgresql not supporting GENERATED column (which is normal)  ,  I would like to know how to tell that this test should only be run on postgres 12+  ? 

(for postgres < 12 , my changes will be tested against  regression by the other test checking for default values , i.e the one just above )

Reply:
found it !